contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -1782,7 +1894,15 @@ PrepareRowDescription(TupleDesc typeinfo, List *targetlist, int16 *formats,
 					 * than -1.
 					 */
 					if (atttypmod == -1 && tle != NULL)
-						atttypmod = resolve_numeric_typmod_from_exp((Node *) tle->expr);
+					{
+						if (!plannedstmt)
+						{
+							ereport(ERROR,
```

Comment:
should test for both, plannedstmt and plannedstmt->planTree both should be not NULL. Also, in error message add context that plan stmt is NULL

Reply:
At the moment, we are not able to create test case which would hit this error.

Reply:
I mean check both plannedstmt and plannedstmt->planTree for error condition